src/_utils/itineraryLocation/index.tsx

```diff
@@ -61,6 +60,12 @@ const StyledItineraryLocation = styled(ItineraryLocation)`
 
   & .kirk-itineraryLocation-label {
     flex: 1;
+    min-width: 0; /* reset flex default to allow shrinking below content width */
+  }
+
+  & .kirk-itineraryLocation-label > p {
```

Comment:
Can we add a class to target the element?

Reply:
Done